src/components/ui/AnswerBoard.js

```diff
@@ -212,4 +213,4 @@ class AnswerBoard extends React.Component {
   }
 }
 
-export default AnswerBoard;
+export default withRouter(AnswerBoard);
```

Comment:
What makes it difference (My wonder)?

Reply:
`withRouter` is for `this.props.history.goBack()'! 
`goBack()` is only available when using `withRoute`